common/src/main/java/com/revenuecat/purchases/common/FileHelper.kt

```diff
@@ -0,0 +1,63 @@
+package com.revenuecat.purchases.common
+
+import android.content.Context
+import java.io.BufferedReader
+import java.io.File
+import java.io.FileInputStream
+import java.io.FileOutputStream
+import java.io.InputStreamReader
+
+class FileHelper(
+    private val applicationContext: Context,
+) {
+    fun appendToFile(filePath: String, contentToAppend: String) {
+        val file = getFileInFilesDir(filePath)
+        file.parentFile?.mkdirs()
+        val outputStream = FileOutputStream(file, true)
```

Comment:
When reading this, I had to google `FileOutputStream` to check the second param to ensure that it meant that the stream would append to the end of file when writing. 

I realize that when you're in android studio, this is probably not a problem since it smartly shows you what each positional argument means, but... maybe it's worth using named params explicitly for this? i.e.: 
```suggestion
        val outputStream = FileOutputStream(file = file, append = true)
```

Reply:
Ahh, `FileOutputStream` is not a Kotlin function so we can't use named parameters. Going to extract to a constant though so we can get the clarity though.